src/modules/metaverse/metaverseOps.ts

```diff
@@ -0,0 +1,92 @@
+/*
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+*/
+
+import axios from "axios";
+
+import { Metaverse } from "@Modules/metaverse";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { Config } from "@Base/config";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import Log from "@Modules/debugging/log";
+
+// Standard response from metavers-server API requests
+export interface APIResponse {
+    status: "success" | "fail";
+    data?: unknown;
+    error?: string;
+}
+
+/**
+ * Return cleaned up URL to the metaverser-server.
```

Comment:
```suggestion
 * Return cleaned up URL to the Metaverse server.
```

Reply:
This typo is in multiple files. Looks like my fingers get confused between "metaverse" and "server".
All occurrences will be fixed.